### DISCUSSION TITLE
Messages pending in the queue

### DISCUSSION THREAD

**Discussion by charlesmudy** [postalserver/postal#1714] [Help with using Postal]
I am having this issue too, over 9K messages are currently in queue and no clue what to do with them. I know there are similar issues reported like this but was unable to find a solution that works for me.

I used this code: Server['yournotify/yournotify'].message_db.messages(:where => {:held => 1, :scope => 'outgoing'}).each { |m| m.add_to_message_queue(:manual => true) } but it seems I have to now manually release them which I don't want.

How to force retry all messages in queue/held mode?

**Comment by willpower232** [comment]
You'll have to do it in code somehow, I think it is mentioned in one of the issues somewhere.
